thasipablo/JavaScript_capstone_project
# Feature: Add Header and Footer

# Changes Introduced:

- Added a main header navigation with a logo and site title.
- Implemented a footer with copyright information.

### Purpose:
- The header provides easy site navigation and branding.
- The footer offers additional information and can be expanded in the future with more links or details.

### Notes:
- The logo in the header might need further adjustments in terms of size and positioning.
- General styles are kept minimal for now and can be updated based on feedback.

### How to Test:
1. Start the development server.
2. Visit the main page and observe the new header and footer.
3. Ensure that they are responsive and appear correctly on different screen sizes.
